netcore/CoreFX.issues.rsp

```diff
@@ -952,15 +952,6 @@
 # https://github.com/mono/mono/issues/15192
 -nomethod System.Reflection.Tests.AssemblyTests.LoadFromStream_Location_IsEmpty
 
-####################################################################
-##  System.Reflection.Metadata.Tests
-####################################################################
-
-# Expected null, got a double byte array
-# https://github.com/mono/mono/issues/15193
```

Comment:
Unrelated change ?

Reply:
yeah, I'll revise all `-nomethod` in a separate PR